src/transformState.js

```diff
@@ -1,11 +1,29 @@
-'use strict';
-
-/**
- * @param {Object} state
- * @param {Object[]} actions
- */
-function transformState(state, actions) {
-  // write code here
-}
-
-module.exports = transformState;
+'use strict';
+
+/**
+ * @param {Object} state
+ * @param {Object[]} actions
+ */
+function transformState(state, actions) {
+  for (const action of actions) {
+    if (action.type === 'addProperties') {

```

Comment:
it would be better to use `switch/case` instead of `if/else`, in case of the limited amount of conditions